src/sentry/middleware/access_log.py

```diff
@@ -74,7 +76,7 @@ def _create_api_access_log(
         user_id = getattr(request_user, "id", None)
         is_app = getattr(request_user, "is_sentry_app", None)
         org_id = getattr(getattr(request, "organization", None), "id", None)
-
+        group = get_rate_limit_config(request.resolver_match.func.view_class).group
```

Comment:
So we don't have to resolve the request path twice, save `request.resolver_match` to a local variable